WalletWasabi/CoinJoin/Client/Clients/AliceClientBase.cs

```diff
@@ -28,10 +28,9 @@ public abstract class AliceClientBase : IDisposable
 			IEnumerable<BitcoinAddress> registeredAddresses,
 			IEnumerable<Requester> requesters,
 			Network network,
-			Func<Uri> baseUriAction,
-			EndPoint torSocks5EndPoint)
+			IHttpClient httpClient)
 		{
-			TorClient = new TorHttpClient(baseUriAction, torSocks5EndPoint, isolateStream: true);
+			TorClient = httpClient;
```

Comment:
It sounds weird. Probably we should rename the `TorClient` property to `HttpClient`.

Reply:
Yes, it should. I'll do it in a follow-up PR.